Core/IntegrationTests/Api.cs

```diff
@@ -108,10 +108,10 @@ public async Task StreamUpdate()
     }
 
     [Test, Order(30)]
-    public async Task StreamGrantPermission()
```

Comment:
tests here will still always fail because `streamUpdatePermission` is not a swap and replace with `streamGrantPermission` as you can't "grant" a role to a new user anymore - they need to be invited.